cluster/cluster-master.yml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: couchbase-master
-        image: arungupta/couchbase
+        image: rsmitty/couchbase
```

Comment:
Is the only difference between `rsmitty/couchbase` and `arungupta/couchbase` a different namespace?